examples/images_contours_and_fields/quiver_head_pos_demo.py

```diff
@@ -0,0 +1,39 @@
+"""
+==================
+Quiver Arrowhead Position Demo
```

Comment:
Please make the heading marker length match the length of the text.

Sphinx warns:
~~~
/home/circleci/project/doc/gallery/images_contours_and_fields/quiver_head_pos_demo.rst:10: WARNING: Title overline too short.
~~~